### ISSUE TITLE
POSIX.1e ACL Support

### ISSUE THREAD

**Issue by dsheets** [stevvooe/continuity#42]
FreeBSD, Linux, and OS X each offer variants of this API. On OS X (at least), the ACL API is separate from extended attributes.

**Comment by stevvooe** [comment]
@dsheets What is the proposal here? Do you want to capture this information?

**Comment by dsheets** [comment]
Yes, ACLs should be recorded in manifests/archives so that they can be persisted.

**Comment by stevvooe** [comment]
Ok, let's figure out a test example and work from there. We should be able to handle this as well as anything!
